Content.Server/DeviceLinking/Components/ObjectSensorComponent.cs

```diff
@@ -0,0 +1,45 @@
+using Content.Server.DeviceLinking.Systems;
+using Content.Shared.DeviceLinking;
+using Content.Shared.Tools;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.DeviceLinking.Components;
+
+/// <summary>
+/// An anchored subfloor thing that detects entities that are on the same tile.
+/// </summary>
+[RegisterComponent, Access(typeof(ObjectSensorSystem))]
+public sealed partial class ObjectSensorComponent : Component
+{
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
```

Comment:
all datafields are automatically rw so just [DataField]

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/45953835/b5b32f28-3ae3-4d05-a025-f5b6718edfac)
no they're not

Reply:
well since VVAccess.ReadWrite apparently doesn't do anything anyway i'll remove it

Reply:
they are its just your enum isnt reflectedor something, and ProtoIds dont support writing ig

Reply:
> they are its just your enum isnt reflectedor something, and ProtoIds dont support writing ig

eh it'll be fine